index.js

```diff
@@ -523,15 +528,6 @@ var compile = function(schema, cache, root, reporter, opts) {
   validate = validate.toFunction(scope)
   validate.errors = null
 
-  validate.__defineGetter__('error', function() {
```

Comment:
If you like I can implement with `defineProperty` and remove the functionality of `validate.error` for IE versions that don't support it?